src/app/pages/dashboard-old/components/widget-sys-info/widget-sys-info.component.ts

```diff
@@ -134,6 +134,11 @@ export class WidgetSysInfoComponent extends WidgetComponent implements OnInit, O
       });
   }
 
+  getDateTimeByTimezone(timeZone: string): number {
```

Comment:
I noticed that this function is not used anywhere else. I'd suggested to reuse existing 
 `<ix-date` component here, if possible

Reply:
> I noticed that this function is not used anywhere else. I'd suggested to reuse existing `<ix-date` component here, if possible

I think we can't use `ix-date` here since we need custom text, not just plain date.
currently we show: `Uptime: 26 minutes 11 seconds as of 02:15 `